.deployment.config.json

```diff
@@ -46,15 +59,6 @@
       {
         "system_certifier": "snyk-code"
       }
-    ],
-    "prd": [
```

Comment:
![Screen Shot 2022-05-05 at 5 20 58 PM](https://user-images.githubusercontent.com/4923043/167135841-0dd3808d-5fa9-495e-8d2c-a45405a81d81.png)
Runs before we have to manually deploy prod, now it will run at the same time as the CDN